mlir/lib/Dialect/Rock/Transforms/BlockwiseGemmToThreadwise.cpp

```diff
@@ -423,6 +457,21 @@ struct BlockwiseGemmAccelRewritePattern
     Value mWavesConstantOp = b.create<ConstantIndexOp>(loc, mWaves);
     Value nWavesConstantOp = b.create<ConstantIndexOp>(loc, nWaves);
 
+    int64_t aCopyPerThread = (K * M * KPack) / op.getBlockSize();
+    int64_t bCopyPerThread = (K * N * KPack) / op.getBlockSize();
+
+    auto maybeCopyAPerThread = computeCopyPerThread(dataTypeA, aCopyPerThread,
```

Comment:
Here also, lets not recompute mPerThread and nPerThread here and pass in as attributes on the basis it was decided when the op is created and it should be upto the caller to decide that.